lib/src/creator_utils.dart

```diff
@@ -25,15 +32,21 @@ class PubspecCreator {
   bool isPrivate;
   String sdkVersion;
   List<DependencyCreator> dependencies;
-  bool createPubspecFile;
```

Comment:
PubspecCreator now always creates a pubspec. To get it to not create one, an empty list of creators is now used instead.

Reply:
smart